cinema/serializers.py

```diff
@@ -35,18 +35,16 @@ class MovieListSerializer(MovieSerializer):
     )
     actors = serializers.StringRelatedField(many=True, read_only=True)
 
-    class Meta:
+    class Meta(MovieSerializer.Meta):
```

Comment:
you don't need to inherit here in Meta. Your serializer inherits from `MovieSerializer` and it is enough for it to have all fields from parent class